openshift/README.rst

```diff
@@ -0,0 +1,29 @@
+# OpenShift Templates
```

Comment:
For .rst files you need to add a heading like: 
```
Openshift Templates
-------------
```

Reply:
Fixed